recipes/stock-pandas/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "1.4.5" %}
+
+package:
+  name: stock-pandas
+  version: {{ version }}
+
+source:
+  url: https://github.com/kaelzhang/stock-pandas/archive/refs/tags/{{ version }}.tar.gz
+  sha256: ec8f3c97fe4a1b60541fc3b059f8fcfa1c288669b0a9e6e8eb27730dac9c4e01
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  script_env:
+  - STOCK_PANDAS_BUILDING_EXT=1
+  number: 0
+
+requirements:
+  build:
+    - {{ stdlib("c") }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - Cython >=3.0.5,<4.0.0
+    - numpy >=2.0.0rc1
+    - pandas >=1.0.0
+    - setuptools
+  run:
+    - python {{ python }}
```

Comment:
I'm unsure what `{{ python }}` is referring to here. I can see that the CI is passing, but I haven't seen that convention before. Did you see this in docs somewhere? Typically it would just be `- python`

Reply:
I guess it's used in a small number of recipes on conda-forge, but I hadn't seen it previously:
https://github.com/search?q=org%3Aconda-forge+%22python+%7B%7B+python+%7D%7D%22&type=code

Reply:
@synapticarbors 

Actually stock-pandas requires python >= 3.9, sorry I am not sure how to define the minimum version in conda recipe.

Reply:
For packages with compiled extensions, we build for python 3.9 - 3.13. Typically if you wanted to restrict the python version you would use the `skip` convention:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#skipping-builds

and it would be something like `skip: True  # [py<39]`. See:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L35